src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/PackageSourceTelemetry.cs

```diff
@@ -209,13 +213,64 @@ public async Task SendTelemetryAsync()
                     sourceRepository = new SourceRepository(new PackageSource(source), Repository.Provider.GetCoreV3());
                 }
 
+                Uri sourceUri = null;
+                bool hasAnHttpResource = false;
+
+                try
+                {
+                    sourceUri = new Uri(source);
+                }
+                catch (ArgumentException)
+                {
+
+                }
+                catch (UriFormatException)
+                {
+
+                }
+                finally
+                {
+                    if (sourceUri != null)
+                    {
+                        if (sourceUri.Scheme == Uri.UriSchemeHttps)
+                        {
+                            NumberOfHTTPSSources++;
+                        }
+                    }
+                }
+
+
+                if (sourceRepository.GetServiceIndexV3FromCache(out ServiceIndexResourceV3 resource))
+                {
+                    sendResourceTelemetry |= true;
+                    var entries = resource.Entries;
+
+                    foreach (var entry in entries)
+                    {
+                        var resourceUri = entry.Uri;
+
+                        if (sourceUri != null && resourceUri.Scheme == Uri.UriSchemeHttp && resourceUri.Scheme != Uri.UriSchemeHttps)
+                        {
+                            hasAnHttpResource |= true;
+                        }
+                    }
+                }
+
+                NumberOfHTTPSSourcesWithHTTPResource += hasAnHttpResource ? 1 : 0;
+
                 var telemetry = await ToTelemetryAsync(data, sourceRepository, parentId, _actionName, _packageSourceMappingConfiguration);
 
                 if (telemetry != null)
                 {
                     TelemetryActivity.EmitTelemetryEvent(telemetry);
                 }
             }
+
+            if (sendResourceTelemetry)
+            {
+                var resourceTelemetry = new ServiceIndexHttpResourcesTelemetry(NumberOfHTTPSSources, NumberOfHTTPSSourcesWithHTTPResource, "ServiceIndexHttpResourcesSummary");
```

Comment:
new telemetry event.

Reply:
If you want to send it in a summary event, there's also some that exist. https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SourceTelemetry.cs. 

It's also worth considering whether it's ok for package source diagnoistics to incldue that information. 

In the end, we'd care about how often sources have this behavior. 
Id' be shocked if it's not 0 tbh.